bookmarks/forms.py

```diff
@@ -0,0 +1,8 @@
+from django import forms
+from bookmarks.models import Bookmark
+
+
+class BookmarkForm(forms.ModelForm):
+    class Meta:
+        model = Bookmark
+        fields = ('url', 'urlname')
```

Comment:
I would think that you would need either the User or Course listed here, depending on if this form is User based or Course based.
(For example, if a user is looking at his bookmarks, it would need to have the course and if you are looking at all the bookmarks for a specific course, you would want to have the username)

Reply:
Course was added into fields.